packages/wells/src/client/model/Wellbore.ts

```diff
@@ -48,12 +48,6 @@ export interface Wellbore {
    * @memberof Wellbore
    */
   id: number;
-  /**
-   * Wellbore description
-   * @type {string}
-   * @memberof Wellbore
-   */
-  description?: string;
```

Comment:
Why have you removed the wellbore description?

Reply:
Couldn't find it in the data class: 
![image](https://user-images.githubusercontent.com/25263842/120166885-60367380-c1fd-11eb-8c37-ea30fc42abbb.png)

Reply:
Make sure you have the latest version of the service, it was only added recently